src/fdk_model_publisher/service/mapper.py

```diff
@@ -72,6 +72,15 @@ def map_model_from_dict(
     model.modelelements = model_element_mapper.extract_model_items(
         partial_model.schema, partial_model.endpoint_description
     )
+
+    if partial_model.format == "JSON":
+        foaf_document = FoafDocument()
+        foaf_document.title = {None: "JSON"}
```

Comment:
Eg forstår ikkje at tittel på documentet = tittel på mediatypen til formatet til dokumentet.
I spekken står det:
> Brukes til å referere til tittel for dokumentet. Bør gjentas når tittelen finnes i flere ulike språk/målformer.

Ref https://data.norge.no/specification/modelldcat-ap-no/#Dokument-tittel

@ToHelland Kan du bekrefte at dette er korrekt? 

Eg hadde forventa at tittel på dokumentet skal vere lik tittel på modellen, sidan hasFormat peiker på ein annan _representajon_ av den _same modellen_.

Reply:
Det stemmer slik som du skriver  @stigbd.

Reply:
Hei dere beklager, da endrer jeg det. Bakgrunnen for å gjøre det slik som jeg foreslo var at jeg oppfattet det slik at det funksjonelle kravet i brukergrensesnittet var at det skulle stå navnet på formatet. Nå kommer det altså til å stå navnet på API'et, siden jeg forstår det slik at navnet på formatet hentes fra til tittel til foafDocument. Men hvis dette er riktig så skal jeg ikke protestere...

Reply:
Vi ønsker at både tittel og format skal vises. Her kan det også være et mangeforhold (o..n), at modeller kan forekomme i flere formater. Tittel er ikke obligatorisk: 
![image](https://user-images.githubusercontent.com/17720749/117409047-03eb7700-af11-11eb-98f5-4bb1fee953cc.png)

Reply:
Jeg skjønner Tore. Jeg endrer koden slik at tittel hentes fra Open-API'en sin tittel slik som @stigbd etterspørr. Men det som gjenstår å avklare da er hvor navnet på formatet skal settes, altså det som dere ser i bildet fra brukergrensesnittet der det står f.eks PNG, XML og i vårt tilfelle her JSON

Reply:
PNG, XML osv. hentes fra EU-vokabularet, som format refererer til.

Reply:
Ja, den er jeg med på. Men vi trenger et felt i modelldcatno der denne informasjonen settes på foafDocument i backend slik at det kan vises i brukergrensesnittet. Jeg satte den i utgangspunktet på foafDocument.title, men der setter vi api'et sin title. En løsning kan være bare å sette f.eks http://publications.europa.eu/resource/authority/file-type/JSON på foafDocument.format. Så kan brukergrensesnittet hente ut JSON som substring fra dette. Er dette ok?

Reply:
Vi kan godt servere brukergrensesnittet litt tekst som den må "mekke" litt på først, trenger ikke å servere alt på fat, for å si det på en folkelig måte, slik som den løsningen jeg først la opp til.

Reply:
JA, det er med at for format på foaf:Document skal EU-vokabularet benyttes. Se eksempel: https://data.norge.no/specification/modelldcat-ap-no/#Skal-brukes-for-format

Reply:
Dersom du går til http://publications.europa.eu/resource/authority/file-type/JSON, vil du sjå at det triplet har skos:prefLabel med verdi "JSON". Så alt fdk-model-publisher treng å gjere er å publisere `dct:format http://publications.europa.eu/resource/authority/file-type/JSON ;` så vil nokon (feks harvester eller nokon andre) hente skos:prefLabel frå http://publications.europa.eu/resource/authority/file-type/JSON.

Reply:
@stigbd Selve dokumentet bør være oppslagsbart. Er det IRI'en for dokumentet som da benyttes? Eller mangler vi noe? Vi bruker jo foaf:Document også for homepage.

Reply:
Hei @stigbd , takk for avklaringen, da gjør vi det sånn!

Reply:
@ToHelland Viss du med dokumentet meiner openAPI-spekken, så bør ein då bruke url til spekken som uri for foaf:Document-instansen (trur eg).

Reply:
@stigbd Ja, det var det jeg mente.

Reply:
Det er vel IRI som er naturlig å bruke.

Reply:
> Det er vel IRI som er naturlig å bruke.
Det forstår eg ikkje. Men greit nok. Kva skal verdien til IRI vere, for eksempel?

Reply:
Ikke helt bra formulert:-) 

Men, vi ønsker at det skal være mulig å slå opp på dokumentet.

foaf:Document er også range for homepage. Den benyttes jo gjerne slik: foaf:homepage  <URL til hjemmeside>. For andre representasjoner (hasFormat) av modellen, ønsker jeg å kunne se nærmere på disse. For OpenAPI-spesifikasjoner ønsker jeg f.eks. å slå opp på dette: https://raw.githubusercontent.com/Informasjonsforvaltning/dsop-api-spesifikasjoner/master/specs/Nordax_Bank_AB_916573154_Accounts-API.json

Men hvor legger vi denne informasjonen? Eneste mulighet, slik spesifikasjonen er nå, er å oppgi dette som en IRI. Alterntivt er at du har en rdfs:seeAlso eller lignende egenskap på foaf:Document hvor du legger denne verdien.

Reply:
Men ser i DCAT-AP-NO er foaf:Document benyttet som range for flere ting, så den problemstillingen (hvis det er noen:-)) gjelder også der.

Reply:
Enig. Feks slik:
```
:my_model a modelldcatno:InformationModel ;
  dct:hasFormat  <https://raw.githubusercontent.com/Informasjonsforvaltning/dsop-api-spesifikasjoner/master/specs/Nordax_Bank_AB_916573154_Accounts-API.json> ;
.

<https://raw.githubusercontent.com/Informasjonsforvaltning/dsop-api-spesifikasjoner/master/specs/Nordax_Bank_AB_916573154_Accounts-API.json> a foaf:Document ; 
    dct:title "My document's title, which probably is identical to the title of the openApi-specification"@en ;
    dct:format <http://publications.europa.eu/resource/authority/file-type/JSON> ;
.
```

Reply:
@frederik-ronnevig  Merk at eksempelet over no er korrekt i forhold til @ToHelland  sin kommentar, som eg ikkje forstod først.

Reply:
> Men ser i DCAT-AP-NO er foaf:Document benyttet som range for flere ting, så den problemstillingen (hvis det er noen:-)) gjelder også der.

Ser ikkje at det er ei utfordring. At ein ting brukes som range for mange ting, er ikkje noko problem.
Utfordringa er når ein bruker url til feks heimeside som iri i foaf:homepage, der denne ikkje har rdf-representasjon. (Men det kunne den ha hatt innebygga, btw.) 
Ved å lage dette som ei blank node, synes eg vi omgår problemet greit.

Reply:
Hei ok, takk for flere tilbakemeldinger! Jeg forstår det da slik at vi setter 

foaf.document.title = openapi.title 

og samtidig

rdfs.see_also = openapi'et.url

Kanskje vi skal ta dette i en egen task som jeg setter opp? Trenger dere mer tid til å avgjøre?

Reply:
Veldig glad for at dere kommenterer og gir tilbakemeldinger! Det betyr at det blir av svært god kvalitet det som blir sluttproduktet av dette!

Reply:
Men i <ein_skolem_uri> i eksemplet ditt, så har du benyttet seeAlso. Denne har vi ikke definert for foaf:Document i spesifikasjonen.

Reply:
Nei, men rdfs-ting bør vere "lov" å bruke over alt.